tests/bson-corpus/int64-valid-004.phpt

```diff
@@ -1,7 +1,5 @@
 --TEST--
 Int64 type: 0
---XFAIL--
-Depends on PHPC-941 and PHP encoding integers as 32-bit if range allows
```

Comment:
I don't recall the original context, but this now passes with PHPC-941 implemented? If so, I suppose "PHP encoding integers as 32-bit if range allows" wasn't really relevant?